sdk/attestation/azure-security-attestation/samples/attestation/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+
+project (attestation-attestation LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+macro (define_sample samplename)
+add_executable (
+  attestation-${samplename}
+  ${samplename}.cpp
+  attestation_collateral.cpp 
+  attestation_collateral.hpp)
+
+create_per_service_target_build(attestation-${samplename} attestation)
+
+target_link_libraries(attestation-${samplename} PRIVATE azure-security-attestation get-env-helper)
```

Comment:
no identity?

Reply:
The sample doesn't use anything from identity. Is there a reason to add it if it's not needed?